GUI.py

```diff
@@ -0,0 +1,297 @@
+from tkinter import *
```

Comment:
Need to add this to requirements.txt

Reply:
Tkinter comes with the latest version of python but I'll add it, thanks

Reply:
Oh shoot, did not realize that. Good to know.

Reply:
I think I should take a serious look, is this function already implemented